package.json

```diff
@@ -97,6 +101,14 @@
       "test/**/*.js"
     ]
   },
+  "tsd": {
```

Comment:
Unfortunately tsd does not support a separate config file. I removed the stuff which is not necessary, as it is defined in tsconfig.json. But the directory attribute can not be changed, because it is a attribute of package.json, which is overridden. 

I extracted the eslint configuration and put it into eslintrc.json